### ISSUE TITLE
ExpectedBucketOwner with s3.to_parquet

### ISSUE THREAD

**Issue by adarsh-chauhan** [awslabs/aws-data-wrangler#562]
Hi All,

How can i use `ExpectedBucketOwner` argument with `s3.to_parquet` method?

**Comment by maxispeicher** [comment]
You can pass additional `kwargs` for S3 with the `s3_additional_kwargs` argument of `s3.to_parquet`. However `awswrangler` performs a check if the keyword is supported and purges it if is not included in https://github.com/awslabs/aws-data-wrangler/blob/main/awswrangler/s3/_fs.py#L27.
Currently `ExpectedBucketOwner` is not included as accepted kwarg but it can be easily added to the accepted kwargs and then it should be possible to use it with `s3_additional_kwargs`.

I've added `ExpectedBucketOwner` as an accepted kwarg on a branch. Can you test if it works for you?
```
pip uninstall awswrangler -y
pip install git+https://github.com/maxispeicher/aws-data-wrangler.git@add-expected-bucket-owner-kwarg
```

**Comment by adarsh-chauhan** [comment]
yes that works

**Comment by igorborgest** [comment]
Released on version [2.5.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/2.5.0)!
